e2e/config/config.go

```diff
@@ -61,7 +61,10 @@ type PVCSpec struct {
 	StorageClassName string `json:"storageClassName"`
 	AccessModes      string `json:"accessModes"`
 }
-
+type Deployer struct {
+	Name string `json:"name"`
+	Type string `json:"type"`
+}
```

Comment:
consider adding a newline before and after the Deployer struct for consistency with surrounding definitions.